datalad/distribution/dataset.py

```diff
@@ -405,7 +404,6 @@ def get_superdataset(self, datalad_only=False, topmost=False,
         -------
         Dataset or None
         """
-        from datalad.coreapi import subdatasets
```

Comment:
I thought we still need those "dummy" imports from `.api` and `.coreapi` to get the methods of those bound to dataset instance.  But it seems no longer be the case:
```
$> python -c 'from datalad.distribution.dataset import Dataset; ds = Dataset("."); print(ds.subdatasets())'
[]
```
and code archaeology lead me to my own PR https://github.com/datalad/datalad/pull/3156 which resolved that - interfaces should be autoimported, thus no need for explicit imports, nice and damn that I forgot. cool ;-)

But note that bugs might resurface since those imports weren't really "unused", and when running a full sweep of tests it would likely to import everything so effects might be hidden as if compared to a singular command or test execution in a new process

Reply:
> But note that bugs might resurface since those imports weren't really "unused", and when running a full sweep of tests it would likely to import everything so effects might be hidden as if compared to a singular command or test execution in a new process

That doesn't sound desirable! Shall I put those back in?

Reply:
nah! we better make it work without them, and #3156 promised that it must work that way! ;-)
if anything - paranoid someone could check if an import(s) was added after that PR merge -- may be commit message shared some special knowledge I am missing, but IMHO if there was any - should have been a comment in the code for the import.

Reply:
Ah, I see the issue. ``test_utils_catched_dataset.py`` relied on ``utils.py`` to import ``opj``, because it imported it from there. Hopefully my push from a few seconds ago resolves this. :crossed_fingers: